pkg/runtime.native.System.IO.Ports/runtime.native.System.IO.Ports.pkgproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
```

Comment:
Could we add comments above each of this skips to say why is it that we need them?

Reply:
I'm actually not sure, I copied the whole project from elsewhere and left it there. I'll try to remove this in a subsequent PR and if not possible I'll add a comment. Let's leave it as is for now since I don't want to block on the comment for waiting on the official build

Reply:
If this is actually not required I would rather remove it now, that way we will be able to catch a possible validation error now that we wouldn't have if we kept these guys. Basically what you are doing here is kind of a skipping all of the testing validation for this package, so I would rather make sure that validation succeeds in order to get us more confidence. If we indeed need these validation skipped, it would be good to understand why, since it might be due to a package authoring problem.